### DISCUSSION TITLE
Sources for Standard Generic ARM Images?

### DISCUSSION THREAD

**Discussion by backslash112** [kairos-io/kairos#1982] [Q&A]
Hi everyone,

I only found the `kairos/kairos-ubuntu-22-lts-arm-rpi-img` on `quay.io` when looking for standard generic ARM images. Does anyone know of other sources?

Thanks in advance!

**Comment by jimmykarily** [comment]
One such image should have been this: https://quay.io/repository/kairos/kairos-opensuse-leap-arm-generic?tab=tags but I don't see 2.4.0 pushed.
Looking at the pipeline: https://github.com/kairos-io/kairos/actions/runs/6172547933/job/16753029475
it seems that is pushed to `quay.io/kairos/kairos-opensuse-leap:v2.4.0-k3svlatest-k3s1` which is wrong and actually overwrote the amd image.

Running the image under amd64 also shows that something is wrong:

```
~ $ docker run -it quay.io/kairos/kairos-opensuse-leap:v2.4.0-k3svlatest-k3s1
WARNING: The requested image's platform (linux/arm64) does not match the detected host platform (linux/amd64/v3) and no specific platform was requested
```

With all the changes in pipelines for 2.4.0 we messed up the names of some artifacts with pipelines overwriting each others results. With 2.4.2 we have made everything explicit and the names are more consistent. This release should be out sometime this week.

You can see what artifacts are going to be produced here: https://github.com/kairos-io/kairos/blob/master/.github/flavors.json

We will add a note on 2.4.0 about the broken images. Sorry for the confusion.

**Reply by mauromorales** [reply]
Probably related to https://github.com/kairos-io/kairos/issues/1981
